client/webpack.config.js

```diff
@@ -55,6 +55,7 @@ const shouldMinify =
 const shouldEmitStats = process.env.EMIT_STATS && process.env.EMIT_STATS !== 'false';
 const shouldShowProgress = process.env.PROGRESS && process.env.PROGRESS !== 'false';
 const shouldEmitStatsWithReasons = process.env.EMIT_STATS === 'withreasons';
+const shouldCheckForDuplicatePackages = process.env.CHECK_PACKAGE_DUPLICATES === 'true';
```

Comment:
I'd call the flag `CHECK_DUPLICATE_PACKAGES` or `DUPLICATE_PACKAGE_CHECKER` -- more in line with the plugin name 🙂